### PR TITLE
Issue #2929527: Finish the first bundle_items add_to_cart field formatter

### PR DESCRIPTION
**Pull request by steveoliver**
See https://www.drupal.org/project/commerce_product_bundle/issues/2929527

### CODE REVIEW

File: config/schema/commerce_product_bundle.schema.yml [hunk 1]
```diff
@@ -38,3 +38,6 @@ field.formatter.settings.commerce_product_bundle_add_to_cart:
     view_mode:
       type: string
       label: 'View mode'
+    combine:
+      type: boolean
+      label: 'Whether to attempt to combine order items containing the same product variation'
```
Comment (olafkarsten) [inline config/schema/commerce_product_bundle.schema.yml]: "Whether to attempt to combine order items containing the same **product bundle**"

File: src/Plugin/Field/FieldFormatter/AddToCartFormatter.php [hunk 2]
```diff
@@ -34,14 +34,12 @@ public static function defaultSettings() {
...
     $form = parent::settingsForm($form, $form_state);
-    // @codingStandardsIgnoreStart
-    // $form['combine'] = [
-    //      '#type' => 'checkbox',
-    //      '#title' => t('Combine order items containing the same product variation.'),
-    //      '#description' => t('The order item type, referenced product variation, and data from fields exposed on the Add to Cart form must all match to combine.'),
-    //      '#default_value' => $this->getSetting('combine'),
-    //    ];
-    // @codingStandardsIgnoreEnd
+    $form['combine'] = [
```
Comment (olafkarsten) [inline src/Plugin/Field/FieldFormatter/AddToCartFormatter.php:37]: I think all occurences of "product variation" needs to be replaced with "product bundle"

**Review by olafkarsten — CHANGES_REQUESTED** [review]
Sorry, Steve :/
Some small changes needed

File: src/Plugin/Field/FieldFormatter/AddToCartFormatter.php (hunk 2 above)
Comment (steveoliver) [reply src/Plugin/Field/FieldFormatter/AddToCartFormatter.php:37]: (Or removed) ;)

**Comment by steveoliver** [comment]
@olafkarsten I think I got them all.

**Comment by olafkarsten** [comment]
done in 8.x-1.x. Thanks Steve.

**Pull request closed without merging**